test/models/repository_test.rb

```diff
@@ -881,6 +883,20 @@ def test_search_content_view_id
       assert_includes repos, @fedora_17_x86_64
     end
 
+    def test_search_description
+      @repo = build(:katello_repository, :fedora_17_el6,
+                     :environment => @library,
+                     :product => katello_products(:fedora),
+                     :description => 'My description',
+                     :content_view_version => @library.default_content_view_version
+                    )
+      assert @repo.valid?
+
+      @repo.save
+      repos = Repository.search_for("description = \"#{@repo.description}\"")
+      assert_includes repos, @repo
```

Comment:
`assert_equal` would be better to ensure it doesn't return other results that it shouldn't be.